README.md

```diff
@@ -5,7 +5,7 @@
 
 The connector supports streaming data from [Google BigQuery](https://cloud.google.com/bigquery/) tables to Apache Flink, 
 and writing results back to BigQuery tables.
-This is done by using the [Flink’s Datastream API](https://nightlies.apache.org/flink/flink-docs-release-1.17/docs/dev/datastream/overview/) 
+This is done by using the [Flink’s Datastream API](https://nightlies.apache.org/flink/flink-docs-release-1.17/docs/dev/datastream/overview/) as well as [Flink's Table API and SQL](https://nightlies.apache.org/flink/flink-docs-release-1.17/docs/dev/table/overview/)
```

Comment:
This is supported via Datastream and Table API ...

Reply:
Fixed.